src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -23,6 +23,7 @@ public class ModelManager implements Model {
     private Deck deck;
     private final UserPrefs userPrefs;
     private FilteredList<Card> filteredDecks;
+    private String selectedDeckName = null; // null when not selected, may want to consider accepting Set<> next time
```

Comment:
Instead of null when not selected, I wonder if we can make use Optional class to deal with Null more graciously?